src/pages/public_registration/LogPanel.tsx

```diff
@@ -224,8 +253,17 @@ function getTicketFilesInfo(ticket: PublishingTicket, registrationFiles: Associa
   );
 
   return {
-    approvedFileNames: publishedFilesOnTicket.map((file) => file.name),
+    approvedFilenames: publishedFilesOnTicket.map((file) => file.name),
+    filenamesForApproval: filesForApprovalOnTicket.map((file) => file.name),
     numberOfUnpublishableFiles: unpublishedFilesOnTicket.length,
     numberOfDeletedFiles: deletedFilesOnTicket.length,
   };
 }
+
+function getUploadedBy(ticket: PublishingTicket, registrationFiles: AssociatedFile[]): string[] {
+  const ticketFiles = registrationFiles.filter((file) => ticket.filesForApproval.includes(file.identifier));
+  const usernames = ticketFiles
+    .flatMap((file) => (file.uploadDetails ? [file.uploadDetails] : []))
+    .map((details) => details.uploadedBy);
+  return [...new Set(usernames)];
+}
```

Comment:
Noen spesiell grunn til at du bruker `function`-declaration istede for arrow-function? Ikke at der har noen betydning, bare nysherrig 😅